sdk/communication/tests.yml

```diff
@@ -26,5 +26,6 @@ extends:
       AZURE_CLIENT_SECRET: $(COMMUNICATION_CLIENT_SECRET)
       AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING: $(communication-livetest-connection-string)
       AZURE_PHONE_NUMBER: $(communication-livetest-phone-number)
-      INCLUDE_PHONENUMBER_LIVE_TESTS: False
+      # SKIP_PHONENUMBER_LIVE_TESTS skips certain phone number tests such as purchase and release
+      SKIP_PHONENUMBER_LIVE_TESTS: TRUE
```

Comment:
should this be True instead of TRUE?

Reply:
Hmm I matched this with Java, which is in all caps